tests/common/snappi/common_helpers.py

```diff
@@ -436,68 +444,112 @@ def config_wred(host_ans, kmin, kmax, pmax, profile=None):
             return False
 
         """ Ensure that Kmin is no larger than Kmax during the update """
-        if kmin > kmin_old:
-            host_ans.shell('sudo ecnconfig -p {} -gmax {}'.format(p, kmax))
-            host_ans.shell('sudo ecnconfig -p {} -gmin {}'.format(p, kmin))
 
+        gmax_cmd = 'sudo ecnconfig -p {} -gmax {}'
+        gmin_cmd = 'sudo ecnconfig -p {} -gmin {}'
+
+        if asic_value != 'None':
+            gmax_cmd = 'sudo ip netns exec %s ecnconfig -p {} -gmax {}' % asic_value
+            gmin_cmd = 'sudo ip netns exec %s ecnconfig -p {} -gmin {}' % asic_value
+            if asic_type == 'broadcom':
+                try:
+                    host_ans.shell('bcmcmd -n {} "BCMSAI credit-watchdog disable"'.format(asic_value))
```

Comment:
We should not disable credit-watchdog here. Instead, we should add it to https://github.com/sonic-net/sonic-mgmt/blob/aa6da9aa65e9ffafa3ef52ef2c573333c15d0d1b/tests/common/snappi/common_helpers.py#L631 or have a new function.

Reply:
this has been resolved